1.1.0/shipment-status.md

```diff
@@ -486,7 +486,7 @@ If all goes well, the company **Pulp** will receive a response like this:
     },
     {
       "id": "678df9d5-ebc5-4f41-a4ab-2cdf1d991deb",
-      "supplierShipmentNumber": "SHP002",
```

Comment:
This change needs to be synchronized with the mock file `1.1.0/mock/samples/shipment-use-case.B.step-2.json`.